wtforms/fields/core.py

```diff
@@ -476,7 +476,7 @@ def __init__(self, label=None, validators=None, coerce=text_type, choices=None,
         self.validate_choice = validate_choice
 
     def iter_choices(self):
-        if isinstance(self.choices[0], (list, tuple)):
+        if self.choices and isinstance(self.choices[0], (list, tuple)):
```

Comment:
I would not do it that way...

If someone forgot to explicitly set the choices (leaving them at ``None``) then the else clause is triggered on rendering. ``zip(None, None)`` then raises ``TypeError: zip argument must support iteration``.

Reply:
On the other hand - I am very impressed by your quick reaction. Thank you very much!

Reply:
Good catch.
Eh, staying home has at least some benefits :)